ehrql/query_language.py

```diff
@@ -30,12 +30,17 @@
 REGISTERED_TYPES = {}
 
 
-class InvalidOperationError(Exception):
+class LanguageError(Exception):
```

Comment:
Could we call this `EhrqlError`?

Reply:
Yeah, I considered that but got put off by the question of how to capitalise it! I think it's the more natural name though, if we can get over my hang-ups.